fbw-common/src/systems/shared/src/ApproachUtils.ts

```diff
@@ -117,7 +118,7 @@ export class ApproachUtils {
         (approach: RawApproach): string;
     } = ApproachUtils.formatLongApproachName;
 
-    public static approachTypeString(type: ApproachType): string {
+    public static approachTypeString(type: ExtendedApproachType): string {
```

Comment:
We shouldn't want ExtendedApproachType here. That is WT specific (has their extensions that MSFS doesn't use). Instead the plain old MSFS ApproachType is good.

Reply:
This is the one enum that actually exists in the MSFS JS code, so we can just use the existing declaration (I've pushed a fix).